pkgs/build-support/cc-wrapper/cc-wrapper.sh

```diff
@@ -79,6 +79,18 @@ if [ "$NIX_ENFORCE_PURITY" = 1 -a -n "$NIX_STORE" ]; then
     params=("${rest[@]}")
 fi
 
+if test -n "@libcxx@"; then
+    NIX_CFLAGS_COMPILE="$NIX_CFLAGS_COMPILE -isystem@libcxx@/include/c++/v1"
```

Comment:
Why is this needed? Libc++ already has a setup hook (`pkgs/development/libraries/libc++/setup-hook.sh`) that sets these flags:
```
export NIX_CFLAGS_COMPILE+=" -isystem @out@/include/c++/v1 -stdlib=libc++"
export NIX_CFLAGS_LINK+=" -stdlib=libc++ -Wl,-rpath,@libcxxabi@/lib"
```
I introduced that on purpose to simplify the (g)cc wrapper.